### PR TITLE
Cancellation page

### LINKED ISSUE #1249: The incident report is not reset when the user clicks on Cancel or Create a new report
## Summary

The incident report is not reset when the user clicks on Cancel or Create a new report

## Steps to reproduce

1. Open the NCFRS webpage
2. Click on Report Now
3. Click on checkbox and continue
4. Choose "By email" and "By phone",  "Today", "Once" and press "Continue" button
5. Choose "Personal Information" on the next screen and then press "Cancel" or "Cancel" button on the next page

Expected Result: The report should be reset
Actual Result: The report is not reset. The info that the user entered is still displayed

It also happened when the user presses the "Create a new report" option on the thank you page

## Unresolved questions

N/A

### LINKED ISSUE #1286: Cancellation intermediate page
## Summary

Cancellation intermediate page. 

## Design Detail

When a user clicks on the cancel report button. They are taken to this page, to confirm they want to delete their inputs and cancel their report. 

Screen Shot 2020-02-05 at 9 21 35 AM

https://www.figma.com/file/R054FzUOrpP13oTbMsQevj/Sprint-5?node-id=567%3A273

## Unresolved questions

There might be some changes to the content in the yellow box. So lets keep this section empty until we figure out the content

### PR DESCRIPTION
**Pull request by amazingphilippe**
Fixes #1286
Fixes #1249 

# Description

Adds an intermediate cancellation page to confirm deletion of local data. 

Actually deletes the date, therefore fixing issue #1249 as well. 

# Checklist:

- [x] I have looked at my code on GitHub and it all looks good (ex: no random commented out code or console.logs)
- [x] I have added and needed tests for my changes (in particular for new screens)
- [x] I have added a comment to any confusing code

# Required followup work

Maybe a test to ensure data is being deleted

### CODE REVIEW

**Comment by sastels** [comment]
This was deployed:
https://rac-prototyp-cancellati-1cqikl.herokuapp.com/

File: f2/src/__tests__/ConfirmCancelPage.test copy.js [hunk 1]
```diff
@@ -0,0 +1,31 @@
+import React from 'react'
```
Comment (sastels) [inline f2/src/__tests__/ConfirmCancelPage.test copy.js]: lol this file needs a better name! "ConfirmCancelPage.test copy.js" indeed.

File: f2/src/ConfirmCancelPage.js [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+                  history.push('/cancelled')
+                }}
+                type="submit"
+                w={{ base: '100%', md: 'auto' }}
+                variantColor="red"
+              >
+                <Trans id="confirmCancelPage.cancelButton" />
+              </Button>
+              <Button
+                onClick={() => history.goBack()}
```
Comment (sastels) [inline f2/src/ConfirmCancelPage.js]: This is an awesome function to have!

File: f2/src/__tests__/ConfirmCancelPage.test copy.js (hunk 1 above)
Comment (amazingphilippe) [reply f2/src/__tests__/ConfirmCancelPage.test copy.js]: 🙄

File: f2/src/ConfirmCancelPage.js (hunk 2 above)
Comment (amazingphilippe) [reply f2/src/ConfirmCancelPage.js]: I wonder if we can leverage this on the backbutton component

**Pull request merged**